### PR TITLE
Support all cli options described in the first version of docs (#11)

### PR DESCRIPTION
**Pull request by kgevorkyan**
### What's done:
* Add options descriptions and short names to the README and Config class

### CODE REVIEW

File: save-core/src/commonMain/kotlin/org/cqfn/save/core/config/SaveConfig.kt [hunk 1]
```diff
@@ -4,19 +4,43 @@ import okio.ExperimentalFileSystem
...
     val baselinePath: Path?,
+    val excludeSuites: List<String>,
+    val includeSuites: List<String>,
+    val language: LanguageType,
+    val testRootPath: Path,
+    val resultOutput: ResultOutputType,
+    val configInheritance: Boolean,
+    val ignoreTechnicalComments: Boolean,
+    val reportDir: Path,
+    val runSingleTest: Path?
```
Comment (petertrr) [inline save-core/src/commonMain/kotlin/org/cqfn/save/core/config/SaveConfig.kt:45]: Do we need a separate flag for this? Maybe trigger single test mode, when `testRootPath` points to a single `*Test` file?
cc @akuleshov7

File: save-core/src/commonMain/kotlin/org/cqfn/save/core/config/SaveConfig.kt [hunk 2]
```diff
@@ -4,19 +4,43 @@ import okio.ExperimentalFileSystem
...
+ * @property quiet do not log anything
  * @property reportType type of generated report with execution results
  * @property baselinePath path to the file with baseline data
+ * @property excludeSuites test suites, which won't be checked
+ * @property includeSuites test suites, only which ones will be checked
+ * @property language language that you are developing analyzer for
+ * @property testRootPath path to directory with tests
+ * @property resultOutput data output stream
+ * @property configInheritance whether configuration files could inherit configurations from the previous level of directories
+ * @property ignoreTechnicalComments should ignore our technical special comments, that we use to describe warnings
```
Comment (petertrr) [inline save-core/src/commonMain/kotlin/org/cqfn/save/core/config/SaveConfig.kt:23]:
```suggestion
 * @property ignoreTechnicalComments if true, ignore technical comments, that SAVE uses to describe warnings, when running tests
```

**Review by petertrr — APPROVED** [review]
Minor suggestions to docs, overall lgtm

File: save-core/src/commonMain/kotlin/org/cqfn/save/core/config/SaveConfig.kt (hunk 1 above)
Comment (kgevorkyan) [reply save-core/src/commonMain/kotlin/org/cqfn/save/core/config/SaveConfig.kt:45]: It was in initial version of config description, so I hold it.
However, it seems, that's really redundant

File: save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt [hunk 3]
```diff
@@ -49,12 +75,82 @@ fun createConfigFromArgs(args: Array<String>): SaveConfig {
...
+        description = "Data output stream",
+    ).default(ResultOutputType.STDOUT)
+
+    val configInheritance by parser.option(
+        ArgType.Boolean,
+        fullName = "config-inheritance",
+        description = "Whether configuration files should inherit configurations from the previous level of directories",
+    ).default(true)
+
+    val ignoreTechnicalComments by parser.option(
```
Comment (akuleshov7) [inline save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt:117]: ignore-save-comments?

File: save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt [hunk 4]
```diff
@@ -49,12 +75,82 @@ fun createConfigFromArgs(args: Array<String>): SaveConfig {
...
+    val testRootPath by parser.option(
+        ArgType.String,
+        fullName = "test-root-path",
+        description = "Path to directory with tests (relative path from place, where save.properties is stored or absolute path)",
+    ).required()
+
+    val resultOutput by parser.option(
+        ArgType.Choice<ResultOutputType>(),
+        fullName = "result-output",
+        shortName = "output",
```
Comment (akuleshov7) [inline save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt:107]: out?

File: save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt [hunk 5]
```diff
@@ -49,12 +75,82 @@ fun createConfigFromArgs(args: Array<String>): SaveConfig {
...
+    val configInheritance by parser.option(
+        ArgType.Boolean,
+        fullName = "config-inheritance",
+        description = "Whether configuration files should inherit configurations from the previous level of directories",
+    ).default(true)
+
+    val ignoreTechnicalComments by parser.option(
+        ArgType.Boolean,
+        fullName = "ignore-technical-comments",
+        shortName = "ignore",
```
Comment (akuleshov7) [inline save-cli/src/commonMain/kotlin/org/cqfn/save/cli/Config.kt:120]: remove short name

**Review by akuleshov7 — APPROVED** [review]
lgtm

**Pull request merged**